### PR TITLE
add github metadata

### PR DESCRIPTION
**Pull request by wtrocki**

### CODE REVIEW

**Comment by wtrocki** [comment]
This somehow did not work. Easier to add repo to config

File: _config.yml [hunk 1]
```diff
@@ -43,6 +43,7 @@ plugins:
   - jekyll-titles-from-headings
   - jekyll-relative-links
   - jekyll-avatar
+  - jekyll-github-metadata
```
Comment (tombentley) [inline _config.yml:46]: This was already added on line 39. TBH I don't really understand the difference between adding in the gem and adding in the `_config.yml`. Are both required?

Comment (wtrocki) [reply _config.yml:46]: It works locally but not on CI/CD.

Comment (wtrocki) [reply _config.yml:46]: As for difference it depends on version of jeckyll but yes. this PR did not made any fix. Sorry

**Pull request merged**